templates/repo/wiki/view.tmpl

```diff
@@ -62,25 +66,72 @@
 		{{end}}
 		<div class="ui {{if .sidebarPresent}}grid equal width{{end}}" style="margin-top: 1rem;">
 			<div class="ui {{if .sidebarPresent}}eleven wide column{{end}} segment markup">
-				{{.content | Str2html}}
+				{{if .EscapeStatus.BadBIDI}}
+					<div class="ui error message wiki-message">
```

Comment:
Here we can see a CSS class `wiki-message`, but this class name are not set for sidebar content and footer content. Is it by purpose?

And maybe we do not need to introduce new class names like `file-message` or `wiki-message`, eg: it can be selected by `.repository .ui.segment.markup ui.message`